test/e2e/network/netpol/kubemanager.go

```diff
@@ -111,9 +114,14 @@ func (k *kubeManager) getPod(ns string, name string) (*v1.Pod, error) {
 	return kubePod, nil
 }
 
-// probeConnectivity execs into a pod and checks its connectivity to another pod..
+// probeConnectivity execs into a pod and checks its connectivity to another pod.
+// Implements the Prober interface.
 func (k *kubeManager) probeConnectivity(nsFrom string, podFrom string, containerFrom string, addrTo string, protocol v1.Protocol, toPort int, timeoutSeconds int) (bool, string, error) {
 	port := strconv.Itoa(toPort)
+	if addrTo == "" {
+		return false, "no IP provided", fmt.Errorf("wtf")
```

Comment:
missing trailing \"

Reply:
add a serious message , kids are watching ;)

Reply:
whoa no idea how that slipped through ha good catch